File: xianguoji-server/src/main/java/com/xianguoji/server/module/shop/entity/Shop.java
package com.xianguoji.server.module.shop.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;

import java.time.LocalDateTime;

@Data
@TableName("shop")
public class Shop {

    @TableId(type = IdType.AUTO)
    private Integer id;
    private String name;
    private String logo;
    private String description;
    private String phone;
    private String address;
    private String businessHours;
    private Integer isOpen;
    private Integer autoAccept;
    private Integer voiceNotify;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;
}

File: user-front/src/api/types/shop.ts
export interface ShopVO {
  id: number;
  name: string;
  logo: string;
  description: string;
  phone: string;
  address: string;
  businessHours: string;
  isOpen: number;
  autoAccept: number;
  voiceNotify: number;
}
export interface PickupPointVO {
  id: number;
  name: string;
  address: string;
  phone: string;
  businessHours: string;
  latitude: number;
  longitude: number;
}
export interface DeliverySettingVO {
  minOrderAmount: string;
  baseFee: string;
  freeAmount: string;
  timeSlots: { label: string; start: string; end: string }[];
}
